erp_tutorial_hnn_core.ipynb

```diff
@@ -342,109 +313,91 @@
   },
   {
    "cell_type": "markdown",
-   "id": "5e74627d",
    "metadata": {},
    "source": [
-    "Let's see what the simulated dipole looks like compared to the experimental dipole"
+    "Let's see what the simulated dipole looks like compared to the experimental dipole. We will also visualize the spike histograms of the drive for easy interpretation."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "be3538e1",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# compare simulated and experimental dipoles\n",
     "fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6), constrained_layout=True)\n",
     "\n",
+    "avg_dpl = average_dipoles(dpls)\n",
     "exp_dpl.plot(ax=axes[0], show=False)\n",
+    "avg_dpl.plot(ax=axes[0], show=False)\n",
     "\n",
-    "# note that there are two ways to plot a dipole: using the Dipole.plot() method and the plot_dipole() function!\n",
-    "plot_dipole(dpls, ax=axes[0], layer='agg', show=False)\n",
-    "average_dipoles(dpls).plot(ax=axes[0], show=False)\n",
-    "axes[0].legend(['experimental', 'sim. trial 1', 'sim. trial 2', 'sim. avg.'])\n",
+    "axes[0].legend(['experimental', 'simulated avgerage'])\n",
     "\n",
     "# driving input spike histogram\n",
-    "fig = net.cell_response.plot_spikes_hist(ax=axes[1],\n",
-    "                                         spike_types=['evprox', 'evdist'])"
+    "net.cell_response.plot_spikes_hist(ax=axes[1]);"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "69d1bd72",
    "metadata": {},
    "source": [
-    "We can also plot the dipole generated by the pyramidal cells in L2/3 and L5, respectively"
+    "We can also plot the dipole generated by the pyramidal cells in L2/3 and L5, respectively. Looking at this plot, we can conclude that the aggregate activity mainly originates from the L5 cells."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "bc11c7c6",
    "metadata": {},
    "outputs": [],
    "source": [
     "# plot layer-specific dipoles\n",
-    "fig, axes = plt.subplots(4, 1, sharex=True, figsize=(6, 6), constrained_layout=True)\n",
+    "fig, axes = plt.subplots(3, 1, sharex=True, figsize=(6, 6), constrained_layout=True, sharey=True)\n",
```

Comment:
I think it's important to do `sharey=True` here, no @rythorpe ? So you can compare across the layers ...